scipy/stats/tests/test_distributions.py

```diff
@@ -1122,6 +1122,19 @@ def test_entropy(self, c, ref):
         assert_allclose(stats.gompertz.entropy(c), ref, rtol=1e-14)
 
 
+class TestFoldNorm:
+
+    # reference values were computed with mpmath with 50 digits of precision
+    # from mpmath import mp
+    # mp.dps = 50
+    # mp.mpf(0.5) * (mp.erf((x - c)/mp.sqrt(2)) + mp.erf((x + c)/mp.sqrt(2)))
```

Comment:
Could (in some cases) be important to ensure that `x` and `c` are `mpf` objects before doing arithmetic between them, but it doesn't happen to affect the results here.